### DISCUSSION TITLE
[Feature Request] Add threshold and/or graph in alert rules view

### DISCUSSION THREAD

**Discussion by jgournet** [grafana/grafana#67018] [Alerting]
In alert rules view, when an alert triggers (=is in "Firing" mode), it is not possible to see what thresholds the alert breached.
The only way to do that is to login and click on "edit".

Also: in "edit" mode, we can see the resulting graph from the query.

=> could those 2 be added when in view mode ?

(this is because not all users have the rights to edit ... so they do not know why the alert is firing)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82726. Feel free to continue the discussion around this there. Thank you!
